docs/appendices/release-notes/unreleased.rst

```diff
@@ -46,3 +46,7 @@ None
 Fixes
 =====
 
+- Fixed case sensitivity of unquoted column names inside ``ON CONFLICT`` clauses.
+
+- Fixed an resiliency issue on snapshot creation while dynamic columns are
```

Comment:
Looks like this slipped in?

Reply:
Isn't it correct to have both fixes in the file until it is released ?

Reply:
Ah my bad, the other bp is not in yet, i messed that up. Thanks for pointing out.